deploy-lighthouse-reth.py

```
@@ -0,0 +1,896 @@
+        else:
+            print("Failed to download genesis release")
+
+    genesis_release = get_github_release(genesis_repository)
+    if genesis_release:
+        download_genesis_release(genesis_release)
+    else:
+        print(f"Failed to retrieve genesis release for {genesis_repository}")
+
+# Initialize sync urls for selected network
+if eth_network == "mainnet":
+    sync_urls = mainnet_sync_urls
```

Ruff (0.8.2)

`mainnet_sync_urls` may be undefined, or defined from star imports

(F405)